packages/components/src/HeaderBar/HeaderBar.test.js

```diff
@@ -152,7 +152,9 @@ describe('HeaderBar', () => {
 			.dive()
 			.find('withI18nextTranslation(Intercom)');
 		expect(intercomTrigger.length).toBe(1);
-		expect(intercomTrigger.prop('className')).toEqual('btn btn-link');
+		expect(intercomTrigger.prop('className')).toEqual(
+			'tc-header-bar-intercom-default-component btn btn-link',
+		);
```

Comment:
```suggestion
		expect(intercomTrigger.prop('className')). toContain(
			'tc-header-bar-intercom-default-component',
		);
```

Reply:
Thanks ! Don't get it why theme() duplicates